service/create/service.go

```diff
@@ -94,17 +94,42 @@ func (s *Service) Boot() {
 	s.bootOnce.Do(func() {
 		s.Config.Logger.Log("info", "Booted cert-operator")
 
-		s.Config.Logger.Log("info", "Test issuing a cert")
-
 		cert := CertificateSpec{
-			ClusterID:        "cert-test",
-			CommonName:       "api.cert-test.giantswarm.io",
-			IPSANs:           []string{"10.0.0.4", "10.0.0.5"},
-			AltNames:         []string{"api.k8s.cert-test.giantswarm.io"},
-			AllowBareDomains: false,
+			ClusterID:  "g8s-cert-test-eu-west-1",
+			CommonName: "api.cluster-test.g8s.eu-west-1.aws.test.private.giantswarm.io",
+			IPSANs:     []string{"10.0.0.4", "10.0.0.5"},
+			AltNames: []string{
+				"kubernetes",
+				"kubernetes.default",
+				"kubernetes.default.svc",
+				"kubernetes.default.svc.cluster.local",
+			},
+			AllowBareDomains: true,
 			TTL:              "720h",
 		}
 
+		s.Config.Logger.Log("info", "Checking for PKI backend")
+		if !s.checkPKIBackend(cert.ClusterID) {
+			err := s.createPKIBackend(cert)
```

Comment:
No I don't have an example and it's good to know that the alternative names shouldn't change often. In that case I'll leave it as is for now rather than hold things up.